File: day3/sample/c/main.c
#include <stdio.h>

void evenOrOdd(int num);

int main(void){
    evenOrOdd(2);
    evenOrOdd(3);
}

void evenOrOdd(int num) {
    if(num % 2 == 0) {
        printf("%d is even\n", num);
    } else {
        printf("%d is odd\n", num);
    }
}

File: day3/sample/rust/Main.rs
fn main(){
    evenOrOdd(2);
    evenOrOdd(3);
}

fn evenOrOdd(num:i32) -> () {
    if(num % 2 == 0) {
        println!("{} is even", num);
    } else {
        println!("{} is odd", num);
    }
}